### PR TITLE
Update testutils.go atom -> ugnot

### PR DESCRIPTION
**Pull request by piux2**

### CODE REVIEW

**Comment by moul** [comment]
Can you fix failing tests?

File: pkgs/sdk/testutils/testutils.go [hunk 1]
```diff
@@ -37,14 +37,15 @@ func (msg *TestMsg) GetSigners() []crypto.Address {
 
 //----------------------------------------
 // Utility Methods
-
+// Fee gas-wanted 5000, gas-fee 1gnot.
 func NewTestFee() std.Fee {
-	return std.NewFee(50000, std.NewCoin("atom", 150))
+
```
Comment (moul) [inline pkgs/sdk/testutils/testutils.go:42]:
```suggestion	
```

Can you try to configure your IDE to run linting tools like gofumpt automatically?

**Review by moul — CHANGES_REQUESTED** [review]
Can you fix the tests please?

**Comment by piux2** [comment]
No need to change it.  close.

**Pull request closed without merging**